src/test/java/homeworks/RenamePlaylistHmw21.java

```diff
@@ -0,0 +1,38 @@
+package homeworks;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RenamePlaylistHmw21 extends MyBaseTest {
+    @Test
+    public void renamePlaylist()  {
+
+        login();
+        doubleClickChoosePlaylist();
+        enterPlaylistName();
+        Assert.assertTrue(doesPlaylistExist());
+    }
+
+    private boolean doesPlaylistExist() {
+        WebElement playlistElement = driver.findElement(By.xpath("//a[text()='Sucre']"));
+        return playlistElement.isDisplayed();
+
+    }
+    private void enterPlaylistName() {
+        WebElement playlistInputField = driver.findElement(By.xpath("//*[contains(text(),'Edit')]"));
+        playlistInputField.sendKeys((Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE)));
+        playlistInputField.sendKeys("Sucre");
+        wait.until(ExpectedConditions.textToBePresentInElement(playlistInputField,"Sucre"));
+        playlistInputField.sendKeys(Keys.ENTER);
+    }
```

Comment:
I refactored you code:

```
  private void enterPlaylistName() {
        WebElement playlistInputField = driver.findElement(By.cssSelector("input[name='name']"));
        
        playlistInputField.sendKeys((Keys.chord(Keys.CONTROL + "a", Keys.BACK_SPACE)));

        playlistInputField.sendKeys("Sucre");

        playlistInputField.sendKeys(Keys.ENTER);
    }
```

![image](https://user-images.githubusercontent.com/104761482/203236964-8c6bf981-5084-4cee-aafe-46594e8db02f.png)